src/modules/app/components/app/app.jsx

```diff
@@ -44,6 +45,9 @@ import parseQuery from "modules/routes/helpers/parse-query";
 import getValue from "utils/get-value";
 
 import {
+  // MARKET,
+  // REPORT,
+  // DISPUTE,
```

Comment:
Any reason we can't remove these instead of commenting them out?

Reply:
i added those comments for the other banners that are going to use that. but i can get rid of them in the meantime.